jupyter_server/extension/application.py

```diff
@@ -287,10 +299,4 @@ def load_jupyter_server_extension(cls, serverapp, argv=None, **kwargs):
         # Create an instance and initialize extension.
         extension = cls()
         extension.initialize(serverapp, argv=argv)
-
-        # Initialize extension template, settings, and handlers.
-        extension._prepare_config()
-        extension._prepare_templates()
-        extension._prepare_settings()
-        extension._prepare_handlers()
         return extension
```

Comment:
If you have to edit the file anyway, could you please add a line break after the last line?